sdk/core/System.ClientModel/src/Internal/Logging/LoggingHandler.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace System.ClientModel.Internal;
+
+internal class LoggingHandler
```

Comment:
Is this the abstraction over ILogger and EventSource?  We may want to give it a different name, and we should add a comment to the class to help folks understand what it is.